config/action.d/helpers-common.conf

```diff
@@ -5,7 +5,7 @@
 #   (printf %%b "Log-excerpt contains 'test':\n"; %(_grep_logs)s; printf %%b "Log-excerpt contains 'test':\n") | mail ...
 #
 _grep_logs = logpath="<logpath>"; grep <grepopts> -E %(_grep_logs_args)s $logpath | <greplimit>
-_grep_logs_args = "(^|[^0-9a-fA-F:])$(echo '<ip>' | sed 's/\./\\./g')([^0-9a-fA-F:]|$)"
+_grep_logs_args = "(^|[^0-9a-fA-F:])$(echo '<ip>' | sed 's/\./\\./g')(:\d{1,5})?([^0-9a-fA-F:]|$)"
```

Comment:
The problem is the IPv6 group separator `::`, so e.g. `::1:1234` where `1234` is port and `::1:1234` as a pure IP are obviously not the same.
The anchoring `([^0-9a-fA-F:]|$)` has previously prevent confusions like this.